l10n_ar_account_withholding_cba/models/account_tax.py

```diff
@@ -0,0 +1,30 @@
+from odoo import fields, models, api
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    ratio = fields.Float(default=100, help="Ratio to apply to tax base amount")
+
+    @api.constrains('ratio')
+    def _check_line_ids_percent(self):
+        """ Check that the total percent is not bigger than 100.0 """
+        for tax in self:
+            if tax.ratio and not (0 < tax.ratio <= 100.0):
+                raise ValidationError(_('The total percentage (%s) should be less or equal to 100!', tax.ratio))
+
+    def get_withholding_vals(self, payment_group):
+        vals = super().get_withholding_vals(payment_group)
+        if self.withholding_type == 'partner_tax' and self.ratio:
+            vals['withholdable_base_amount'] *= self.ratio / 100
+            vals['period_withholding_amount'] *= self.ratio / 100
+        return vals
+
+    def _compute_amount(
+            self, base_amount, price_unit, quantity, product, partner=None, fixed_multiplicator=1):
+        res = super()._compute_amount(base_amount, price_unit, quantity, product, partner=partner, fixed_multiplicator=fixed_multiplicator)
```

Comment:
you do not change result of the super, so i think you can remove it here and simple call it at the return